app/assets/javascripts/admin/addon/templates/api-keys-index.hbs

```diff
@@ -14,13 +14,13 @@
         <th>{{i18n "admin.api.user"}}</th>
         <th>{{i18n "admin.api.created"}}</th>
         <th>{{i18n "admin.api.last_used"}}</th>
+        <th>{{i18n "admin.site_settings.table_column_heading.status"}}</th>
```

Comment:
should this string be moved to `admin.api.*` like the other table header strings here? or is it the other way around - this is the desired path and the other strings will be moved later? 🙂